templates/reader.html.tt2

```diff
@@ -280,7 +231,7 @@
 
     <div class="pagecount">
         <span class="current-page"></span> /
-        <span class="max-page"></span>
+        <span class="max-page">[% pagecount %]</span>
```

Comment:
TBH I think it's worth keeping it for custom stylesheet purposes. Ideally every part of the page should have its own selectors so that an user can hide/style each of them to their liking.

Reply:
Fair enough, not like it costs much to keep it. 👍